File: practices/old_enough.py
#DG 6th Old enough LaRose
name = input("what is your name\n").title().strip()

age = int(input("how old are you\n"))
if age >= 100:
    print(f"wow {name} you are old enough to be in the museum")
elif age >= 18:
    print("you can vote. be carfull its a big world out there")
elif age >= 16:
    print("you can drive by your self. be freeee")
elif age >= 15:
    print(" you can get a learners permit. spread your wings and dont crash")
elif age >= 4:
    print("you can go to school. the road is open now its your turn to choose your path")
else:
    print(f"{name} you should be using your imagination >:(")

File: c_programming/old_enough.c
//DG 6th old enough
#include <stdio.h>
#include <string.h>
int main(void) {
    char Age;
    char name[50];
    printf("what is your name:\n ");
    scanf("%49s",name);
    printf("welcome %s",name);
    printf(" how old are you.\n");
    scanf("%d", &Age);
    if(Age >= 100) {
        printf("your old enough to be in a museum!!!\n");
    } else if(Age >= 18) {
        printf("you can vote. good luck %s\n",name);
    } else if(Age >= 17) {
        printf("you can drive. dont crash!!\n");
    } else if(Age >= 15) {
        printf("you can get a drivers permit. spread you wings. dont crash!!!!!!!!!!!!\n");
    } else if(Age >= 5) {
        printf("you can go to school. Education is what remains after one has forgotten what one has learned in school.—Albert Einstein\n");
    } else {
        printf("GET OFF THE COMPUTER!!!!!!!");
    } 
    return 0;

}